src/west/_bootstrap/main.py

```diff
@@ -109,74 +120,97 @@ def init(argv):
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        description=
+'''
+Initializes a Zephyr installation.
+
+In more detail, does the following:
+
+  1. Clones the manifest repository to west/manifest, and the west repository
+     to west/west
+
+  2. Creates a marker file west/{}
+
+  3. Creates an initial configuration file west/config
+
+As an alternative to manually editing west/config, 'west init' can be rerun on
+an already initialized West instance to update configuration settings. Only
+explicitly passed configuration values (e.g. --mr MANIFEST_REVISION) are
+updated.
+
+Updating the manifest URL or revision via 'west init' automatically runs 'west
+update --reset-manifest --reset-projects' afterwards to reset the manifest to
+the new revision, and all projects to their new manifest revisions.
+
+Updating the west URL or revision also runs 'west update --reset-west'.
+
+To suppress the reset of the manifest, west, and projects, pass --no-reset to
+'west init'. With --no-reset, only the configuration file will be updated, and
+you will have to handle any resetting yourself.
+'''.format(WEST_MARKER))
+
     init_parser.add_argument(
         '-b', '--base-url',
-        help='''Base URL for both 'manifest' and 'zephyr' repositories; cannot
-        be given if either -m or -w are''')
+        help='''Base URL for both 'manifest' and 'zephyr' repositories. Cannot
+             be given if either -m or -w are''')
+
     init_parser.add_argument(
         '-m', '--manifest-url',
-        help='Zephyr manifest fetch URL, default ' + MANIFEST_DEFAULT)
+        help='Manifest repository URL (or remote name) (default: {})'
+             .format(MANIFEST_URL_DEFAULT))
+
     init_parser.add_argument(
-        '--mr', '--manifest-rev', default=MANIFEST_REV_DEFAULT,
-        dest='manifest_rev',
-        help='Manifest revision to fetch, default ' + MANIFEST_REV_DEFAULT)
+        '--mr', '--manifest-rev', dest='manifest_rev',
+        help='Manifest revision to fetch (default: {})'
+             .format(MANIFEST_REV_DEFAULT))
+
     init_parser.add_argument(
         '-w', '--west-url',
-        help='West fetch URL, default ' + WEST_DEFAULT)
+        help='West repository URL (or remote name) (default: {})'
+             .format(WEST_URL_DEFAULT))
+
     init_parser.add_argument(
-        '--wr', '--west-rev', default=WEST_REV_DEFAULT, dest='west_rev',
-        help='West revision to fetch, default ' + WEST_REV_DEFAULT)
+        '--wr', '--west-rev', dest='west_rev',
+        help='West revision to fetch (default: {})'
+             .format(WEST_REV_DEFAULT))
+
     init_parser.add_argument(
-        'directory', nargs='?', default=None,
-        help='Initializes in this directory, creating it if necessary')
+        '--nr', '--no-reset', dest='reset', action='store_false',
+        help='''Suppress automatic reset of the manifest, west, and project
+             repositories when rerunning 'west init' to update the
```

Comment:
What about "when rerunning 'west init' in an existing installation"?

Reply:
Yeah, that sounds better. Changed.